client/src/pages/collections/Create.tsx

```diff
@@ -53,6 +63,8 @@ const CreateCollection: FC<CollectionCreateProps> = ({ handleCreate }) => {
     autoID: true,
   });
 
+  const [consistencyLevel, setConsistencyLevel] = useState(1);
```

Comment:
we have enum for this value, so you can code useState< ConsistencyLevelEnum >(ConsistencyLevelEnum.session) is more clear

Reply:
Thank you for the comments and I modified as requested. Please take a look again.